Makefile

```diff
@@ -766,6 +773,14 @@ testlogic: TESTFLAGS := -test.v $(if $(FILES),-show-sql)
 testlogic: bin/logictest.test
 	cd pkg/sql/logictest && logictest.test -test.run "$(TESTS)" -test.timeout $(TESTTIMEOUT) $(TESTFLAGS)
 
+# Run make testccllogic to run all of the CCL logic tests. Specify test files to run
+# with make testccllogic FILES="foo bar".
+testccllogic: ## Run SQL Logic Tests.
```

Comment:
`testlogicccl` would fit better with our naming convention, but it awkward to type/say/read.